TRADEMARKS.md

```diff
@@ -30,6 +30,9 @@ The Commonhaus Foundation uses the following unregistered trademarks:
 - OpenRewrite &trade; [^1]
 - SDKMAN! &trade; [^1]
 - Feign &trade; [^1]
+- Quarkus &trade; [^1]
+
+[^1]: Trademark is used by permission subject to the foundation's [bootstrapping agreement](
```

Comment:
I think this footnote line was added by mistake, could you remove it? Thanks

Reply:
oh! right. This doesn't need to be here (as it is at the bottom).
```suggestion
```